kernelci.org/content/en/docs/org/tsc/votes.md

```diff
@@ -35,6 +37,23 @@ outcome and a more detailed section listing each individual vote.
 
 ## Votes
 
+### 2024-03-14
+
+**Motion: Add TSC Member: Shuah Khan**
+* Result: approved
+* Effective date: 2024-03-18
+* Voting method: TSC monthly
+* Voting members: arisut, broonie, gtucker, nuclearcat, pawiecz, spbnick
+* Vote count: 6/6/8 in favour
+
+**Motion: Elect TSC Chair**
+* Candidates: Shuah Khan, Nikolai Kondrashov
+* Result: approved
```

Comment:
`Result: Nikolai elected` instead ?

Reply:
The motion to elect a new char was approved so this is correct afaict.  There's a line with the vote counts for each candidate, it's a bit of a special case as it's not a binary yes / no vote.

Reply:
I could add a line with "Elected: Nikolai Kondrashov" if it's not obvious enough from the vote counts.